### PR TITLE
gateway_c containerization

### PR DESCRIPTION
**Pull request by ymasmoudi**
Signed-off-by: YOUSSEF EL MASMOUDI <ymasmoudi@fb.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/build/c/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+ENV MAGMA_ROOT /magma
+ENV C_BUILD /build/c
+ENV OAI_BUILD $C_BUILD/oai
+
+ENV CCACHE_DIR $MAGMA_ROOT/.cache/gateway/ccache
+ENV MAGMA_DEV_MODE 1
+ENV XDG_CACHE_HOME $MAGMA_ROOT/.cache
+
+# TODO(fixup this include in source code @ MConfigLoader.cpp)
+RUN ln -s /usr/include/nlohmann/json.hpp /usr/include/json.hpp
```
Comment (themarwhal) [inline lte/gateway/build/c/Dockerfile:92]: We should no longer need this line :) https://github.com/magma/magma/pull/8285

**Comment by ekowtaylor** [comment]
@ymasmoudi planning to work on this still?

File: lte/gateway/build/c/Dockerfile (hunk 1 above)
Comment (ymasmoudi) [reply lte/gateway/build/c/Dockerfile:92]: thanks :) actually these changes were merged in #8424
I will update it there and close this one.

**Comment by ymasmoudi** [comment]
> @ymasmoudi planning to work on this still?

we already merged these changes in #8424. 
Let me close it.

**Pull request closed without merging**